circle.yml

```diff
@@ -0,0 +1,14 @@
+dependencies:
+  cache_directories:
+    - "~/.stack"
+  pre:
```

Comment:
Cool. I assumed that downloading Stack would take more than a couple seconds, but caching isn't worth it given your example.